app/views/users/show.html.erb

```diff
@@ -8,4 +8,13 @@
       </h1>
     </section>
   </aside>
+  <div class="col-md-8">
+    <% if @user.microposts.any? %>
+      <h3><%= t ".microposts" %>(<%= @user.microposts.count %>)</h3>
```

Comment:
nó không lấy các microposts từ CSDL.nó thực hiện tính toán trong CSDL.yêu cầu CSDL đếm các microposts với user_id 
gọi chiều dài trên mảng bằng hàm count() 
cái nay trở nên ko hiệu quả nếu microposts lớn